src/analysis/analysis-poller.js

```diff
@@ -0,0 +1,59 @@
+var _ = require('underscore');
+var BackbonePoller = require('backbone-poller');
+
+var AnalysisPoller = {
```

Comment:
I'm feeling wary about having this object as a global/singleton, since separated vis object on the same page will share the cache and thus interfere with each other on poll/reset calls. What's the reason for not having these methods on a function prototype and create an instance when the vis is created instead?